### ISSUE TITLE
KData class

### ISSUE THREAD

**Issue by ckolbPTB** [PTB-MR/mrpro#1]
Dataclass for k-space data

```
class KData():
    KData.header -> ismrmrd header provided by pyismrmrd module
    KData.traj -> torch.Tensor, 5D: [everything_else, 3, k2, k1, k0]
    KData.data -> torch.Tensor, 5D: [everything_else, coils, k2, k1, k0]
```

Check list:
- [x]     Create branch
- [x]     Implement code
- [x]     Add comments
- [x]     Implement tests
- [x]     Squash commmits
- [x]     Create PR
- [x]     Assign reviewer(s) to PR

**Comment by ckolbPTB** [comment]
https://github.com/ckolbPTB/mrpro/tree/kdata

**Comment by fzimmermann89** [comment]
Closed by #29
